modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/build/static/js/main.js

```diff
@@ -1 +1 @@
teElement("div",{className:"site-alert__content footer-alert__content"},l()(this.props.description))),this.props.linkTitle&&a.default.createElement("div",{className:"col-xs-12 col-sm-5 col-md-5 col-lg-5 site-alert__cta"},a.default.createElement("div",{className:"field-alert-link"},a.default.createElement("a",{href:this.props.linkUrl,style:o},this.props.linkTitle))),a.default.createElement("a",{href:"#close",className:"site-alert__dismiss",onClick:function(){return n()},"aria-label":"Close alert",onFocus:function(){return r()}},a.default.createElement("i",{className:"fa fa-times","aria-hidden":"true"},a.default.createElement("span",{className:"visually-hidden"},"Close alert ",l()(this.props.label))))))))}}]),e}(a.Component),v=function(t){return{closeAlert:function(e){t(Object(f.b)(e))}}},y=function(t){return{alerts:t.init.alerts}},m=Object(p.a)(y,v)(h);e.a=m},function(t,e,n){function r(t){return n(o(t))}function o(t){var e=i[t];if(!(e+1))throw new Error("Cannot find module '"+t+"'.");return e}var i={"./store":111,"./store.dev":170,"./store.dev.js":170,"./store.js":111,"./store.prod":173,"./store.prod.js":173};r.keys=function(){return Object.keys(i)},r.resolve=o,t.exports=r,r.id=451},function(t,e,n){"use strict";function r(){var t=arguments.length>0&&void 0!==arguments[0]?arguments[0]:u,e=arguments[1],n=0,r=0;switch(e.type){case a.a:return Object.assign({},t,{loading:!0,error:null});case a.c:var o=e.payload,c=i.a.load("alerts_dismiss");if(c){var l=o.header.local.filter(function(t){return!c.includes(parseInt(t.id))}),f=o.footer.local.filter(function(t){return!c.includes(parseInt(t.id))});o.header.local=l,o.footer.local=f}return"undefined"!==typeof o.header&&"undefined"!==typeof o.header.global&&(n+=o.header.global.length),"undefined"!==typeof o.header&&"undefined"!==typeof o.header.local&&(n+=o.header.local.length),"undefined"!==typeof o.footer&&"undefined"!==typeof o.footer.global&&(r+=o.footer.global.length),"undefined"!==typeof o.footer&&"undefined"!==typeof o.footer.local&&(r+=o.footer.local.length),Object.assign({},t,{loading:!1,alerts:o,headerPager:n>=2,footerPager:r>=2});case a.b:return Object.assign({},t,{loading:!1,error:e.payload});case s.a:var p={},d={},h={},v={},y={};return t.alerts.hasOwnProperty("header")&&(t.alerts.header.hasOwnProperty("global")&&(d=Object.keys(t.alerts.header.global).map(function(n){return parseInt(t.alerts.header.global[n].id)===parseInt(e.payload)?null:t.alerts.header.global[n]}),"undefined"===typeof p.header?p.header={global:d.filter(Boolean)}:p.header.global=d.filter(Boolean)),t.alerts.header.hasOwnProperty("local")&&(h=Object.keys(t.alerts.header.local).map(function(n){return parseInt(t.alerts.header.local[n].id)===parseInt(e.payload)?null:t.alerts.header.local[n]}),"undefined"===typeof p.header?p.header={local:h.filter(Boolean)}:p.header.local=h.filter(Boolean))),t.alerts.hasOwnProperty("footer")&&(t.alerts.footer.hasOwnProperty("global")&&(v=Object.keys(t.alerts.footer.global).map(function(n){return parseInt(
```

Comment:
Sorry, we are not accepting changes to compiled file to distribution. 
Please, add a change to source javascript and jeep it with PR.

Thank you

Reply:
I am not able to understand what are you trying to say. I have made changes with main.js file and I was facing carousel issue with alerts, So I made this changes. No idea about source javascript.

Reply:
@siddharth28192
See https://github.com/ymcatwincities/openy/tree/8.x-2.x/modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts
There is a yarn/npm workflow for the alerts, built by using ReactJS
We have list of commands in here https://github.com/ymcatwincities/openy/blob/8.x-2.x/modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/package.json#L73
It's a modules application https://github.com/ymcatwincities/openy/tree/8.x-2.x/modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/src
WHich contains components and all needed dependencies that should be updated in order to accomplish what you are trying to do in this PR

I'm asking you not to change compiled file, but to change sources, run `yarn build` , get new main.js and add changed sources to this PR.

Reply:
@podarok Thanks for letting me know, how to rebuild main.js file.

Reply:
Thank YOU!